frontend/components/FilterContent.vue

```diff
@@ -85,29 +97,12 @@ export default {
     },
   },
   mounted: function () {
-    const tags = document.querySelectorAll(".tag");
-    const arr = [];
-
-    for (let i = 0; i < tags.length; i++) {
-      tags[i].addEventListener("click", function () {
-        if (arr.includes(tags[i].firstElementChild.innerText)) {
-          const index = arr.indexOf(tags[i].firstElementChild.innerText);
-          if (index > -1) {
-            arr.splice(index, 1);
-          }
-        } else {
-          arr.push(tags[i].firstElementChild.innerText);
-        }
-        tags[i].classList.toggle("highlight");
-      });
-    }
     this.eventBus.on("reset", function (reset) {
       if (reset) {
         let high = document.querySelectorAll(".highlight");
```

Comment:
Why `querySelector` and other native JS methods are used? Since you're using Vue it's antipattern.

You can just clear `activeTags` array, and create computed property which checks if tag is in this array, if it's true, apply one class, if not, apply other.